debug/debug.go

```diff
@@ -17,20 +17,32 @@ import (
 	"github.com/openshift-kni/commatrix/client"
 )
 
+type DebugPodInterface interface {
```

Comment:
I would argue that more things in Go need to pass around interfaces instead of raw structs to allow for better testing in the future (if needed).  This seems fine to me.